vrp-cli/src/lib.rs

```diff
@@ -377,7 +379,7 @@ mod py_interop {
 
     /// Validates and solves Vehicle Routing Problem.
     #[pyfunction]
-    fn solve_pragmatic(problem: String, matrices: Vec<String>, config: String) -> PyResult<String> {
+    fn solve_pragmatic(problem: String, matrices: Vec<String>, config: String, geojson: bool) -> PyResult<String> {
```

Comment:
This requires the change here https://github.com/reinterpretcat/vrp/blob/master/examples/python-interop/example.py#L116

Reply:
https://github.com/reinterpretcat/vrp/pull/108/commits/c8bd7e87770442031962e7c1b20145bc8dea2352